org.clojure/clojure/1.4.0/clj/clojure.core/some/notes.md

```diff
@@ -0,0 +1,14 @@
+## Arities
+
+    [pred coll]
+
+## Docs
+
+Returns the first logical true value of `(pred x)` for any `x` in
+`coll`, otherwise returns nil.
+
+## Notes
+
+Note that `pred` need not return a boolean, merely a truthy value. An
+idiom is to use complex functions as `pred` and then consider the
```

Comment:
idiom is to use complex functions as a 'pred'icate and use the result directly, rather than testing for the value, and then computing a value from that.